client/driver/executor/executor.go

```diff
@@ -901,3 +902,24 @@ func (e *UniversalExecutor) Signal(s os.Signal) error {
 
 	return nil
 }
+
+func GetChrootEnvTree(chrootEnv map[string]string) *iradix.Tree {
```

Comment:
On master as of #2132 all chroot building code has moved out of the drivers+executor and into TaskRunner.